kalite/distributed/management/commands/initialize_kalite.py

```diff
@@ -22,17 +22,20 @@ class Command(BaseCommand):
     )
 
     def setup_server_if_needed(self):
-        """Run the setup command, if necessary."""
-        
-        # Ensure that the database has been synced and a Device has been created
+        """Run the setup command, if necessary.
+            It's necessary if the Settings model doesn't have a "database_version" or if that version doesn't match
+            kalite.version.VERSION, indicating the source has been changed. Then setup is run to create/migrate the db.
+        """
+
         try:
-            assert Settings.get("private_key") and Device.objects.count()
+            from kalite.version import VERSION
```

Comment:
it will prompt database migrations on every minor upgrade, it's not harmful, no, and maybe we will want this feature in alpha releases anyways